internal/tss/tss.go

```diff
@@ -0,0 +1,39 @@
+package tss
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+	"github.com/vultisig/mobile-tss-lib/tss"
+
+	"github.com/vultisig/vultisigner/internal/logging"
+	"github.com/vultisig/vultisigner/relay"
+)
+
+func CreateTSSService(serverURL, Session, HexEncryptionKey string, localStateAccessor tss.LocalStateAccessor) (tss.Service, error) {
+	messenger := &relay.MessengerImp{
+		Server:           serverURL,
+		SessionID:        Session,
+		HexEncryptionKey: HexEncryptionKey,
+	}
+
+	tssService, err := tss.NewService(messenger, localStateAccessor, true)
```

Comment:
the last parameter for `NewService` is whether to create preParams , which is a very time consuming process , it is only required for Keygen,
for keysign, the last paramter should be `false`